cli/createTemplate.sh

```diff
@@ -978,7 +981,7 @@ function process_template() {
     process_template_pass \
       "${entrance}" \
       "${flows}" \
-      "${task_parameters[@]}" \
+      "${task_parameters[@]:0:7}" \
```

Comment:
IS this related to credentials? Looks some insurance against too many parameters being provided?

Reply:
Sorry it snuck in and forgot to mention it. Yes I've been working on extending the generation contract to include more details which means that for now we only want particular values